taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
 
-# Create your views here.
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_cars = Car.objects.count()
+    num_Manufacturers = Manufacturer.objects.count()
+
+    context = {
+    "num_drivers": num_drivers,
+    "num_cars": num_cars,
+    "num_Manufacturers": num_Manufacturers,
```

Comment:
The same naming issue here in the context dictionary. Rename 'num_Manufacturers' to 'num_manufacturers'.